packages/devtools_app/lib/src/timeline/timeline_model.dart

```diff
@@ -931,6 +933,18 @@ class AsyncTimelineEvent extends TimelineEvent {
 
   bool _hasOverlappingChildren;
 
+  bool deepOverlaps(TimelineEvent other) {
```

Comment:
instead of "deepOverlaps" how about isSubtreeOverlapping` or 
`isSubtreeOverlap`

Reply:
high level, how often is this method called? Add a comment warning it may be somewhat expensive.